--- xfdesktop-4.0.6/configure.ac.orig	2004-07-09 10:10:54.000000000 +0200
+++ xfdesktop-4.0.6/configure.ac	2004-07-12 17:13:52.341234304 +0200
@@ -27,8 +27,8 @@
 AC_CHECK_FUNCS(sigaction srandom)
 
 dnl Check for i18n support
-BM_I18N([xfdesktop], [ar az bg ca cs de es es_MX et eu fa_IR fi fr hi hu it ja pl
-	 ko lt ms nl no pl pt_BR pt_PT ru sv ta tr uk vi zh_CN zh_TW])
+BM_I18N([xfdesktop], [ar az bg ca cs de es es_MX et eu fa fi fr hi hu it ja pl
+	 ko lt ms nl nb pl pt_BR pt ru sv ta tr uk vi zh_CN zh_TW])
 
 dnl Check for X11 installed
 BM_LIBX11_REQUIRE
