--- xfdesktop-4.6.1/src/xfce-desktop-menu.c~	2009-01-12 23:03:12.000000000 +0100
+++ xfdesktop-4.6.1/src/xfce-desktop-menu.c	2010-05-16 20:19:02.820729427 +0200
@@ -56,6 +56,22 @@
     }
     
     all_dirs = xfce_resource_lookup_all(XFCE_RESOURCE_CONFIG,
+                                        "menus/applications.menu");
+    for(i = 0; all_dirs[i]; i++) {
+        DBG("checking %s", all_dirs[i]);
+        if(user_menu || strstr(all_dirs[i], userhome) != all_dirs[i]) {
+            if(g_file_test(all_dirs[i], G_FILE_TEST_IS_REGULAR)) {
+                menu_file = g_strdup(all_dirs[i]);
+                break;
+            }
+        }
+    }
+    g_strfreev(all_dirs);
+
+    if(menu_file)
+        return menu_file;
+
+    all_dirs = xfce_resource_lookup_all(XFCE_RESOURCE_CONFIG,
                                         "menus/xfce-applications.menu");
     for(i = 0; all_dirs[i]; i++) {
         DBG("checking %s", all_dirs[i]);
