diff -urN xfdesktop-4.2.2/Makefile.am fix/xfdesktop-4.2.2/Makefile.am
--- xfdesktop-4.2.2/Makefile.am	2005-05-15 10:27:45.000000000 +0200
+++ fix/xfdesktop-4.2.2/Makefile.am	2005-05-20 17:12:57.000000000 +0200
@@ -45,6 +45,7 @@
 	menu.xml.ja							\
 	menu.xml.ko							\
 	menu.xml.nl  							\
+	menu.xml.pl							\
 	menu.xml.pt_BR							\
 	menu.xml.ro							\
 	menu.xml.ru							\
diff -urN xfdesktop-4.2.2/menu.xml.pl fix/xfdesktop-4.2.2/menu.xml.pl
--- xfdesktop-4.2.2/menu.xml.pl	1970-01-01 01:00:00.000000000 +0100
+++ fix/xfdesktop-4.2.2/menu.xml.pl	2005-05-20 17:09:24.000000000 +0200
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE xfdesktop-menu>
+
+<!-- 
+     Opis pliku menu do XFce 4: 
+     =================================
+     Tutaj postaramy się wyjaśnić format pliku menu. Spójrz na menu poniżej
+     by zobaczyć jak to wygląda w praktyce.  Prawde mówiąc nie powinieneś
+     mieć potrzeby edytowania tego pliku ręcznie; sprawdź wzamian
+     xfce4-menueditor.
+-->
+
+<!-- Oczywiście w ten sposób robi się komentarze ;-) -->
+
+<!-- 
+
+   Wszystko musi znajdować sie pomiędzy dokładnie jedną parą 
+    <xfdesktop-menu></xfdesktop-menu> tagów.
+
+   + Aplikacje:
+    <app name="Nazwa w menu" cmd="Komenda do uruchomienia" term="false"
+         icon="ikona"  snotify="false" visible="true" />
+    Atrybut 'term' określa czy program wymaga terminala do działania
+    a 'snotify' ustala czy program obsługuje informowanie o starcie.
+    Możesz wskazać ikonę do wyświetlenia za pomocą atrubutu 'icon'.
+    Tylko 'name i 'cmd' sa wymagane.
+
+   + Separatory:
+    <separator visible="true" />
+    Tworzy poziomy separator.  Atrybut 'visible' powinien być ustawiony.
+
+   + Podmenu:
+    <menu name="Nazwa w menu" icon="ikona" visible="true"></menu>
+    Tylko 'name' jest wymagane, ale możesz także ustawić ikonę do
+    wyświetlania za pomocą atrybutu 'icon'. Pomiędzy tagami menu możesz
+    zdefiniować więcej apliakcji, separatorów i innych menu.
+
+   + Tytuły:
+    <title name="Nazwa w menu" icon="ikona" visible="true" />
+    Tworzy tytuł menu.  Możesz także dodać opconalną ikonę jeśli masz na
+    to ochotę.
+    
+   + Zawieranie innych plików:
+    <include type="file" src="menu2.xml" visible="true" />
+    Zawiera plik menu2.xml.  Względna ścieżka zakłada, że plik jest
+    umiejscowiony w ~/.xfce4/.  Bezwględna ścieżka bedzie traktowana
+    normalnie.
+
+   + Zawieranie autogenerowanego menu:
+    <include type="system" style="simple" unique="true" visible="true" />
+    Zawiera menu systemowe.  'style' może być "simple" lub "multilevel".
+    Jeśli 'unique' jest ustawione na "false", wielokrotne wpisy menu będą
+    generowane jeśli aplikacja pojawia sie w wielu kategoriach.
+-->
+
+<xfdesktop-menu>
+
+	<title name="Menu Pulpitu" icon="xfce4-backdrop"/>
+
+	<separator/>
+
+	<app name="Uruchom Program" cmd="xfrun4" icon="gnome-fs-executable"/>
+
+	<separator/>
+
+	<app name="Terminal" cmd="xfterm4" icon="gnome-terminal"/>
+	<app name="Menadżer plików (xffm)" cmd="xffm" icon="file-manager"/>
+	<app name="Przeglądarka WWW" cmd="mozilla" icon="gnome-globe"/>
+
+	<separator/>
+
+	<menu name="Ustawienia" icon="gnome-settings">
+		<app name="Menadżer ustawień" cmd="xfce-setting-show" icon="gnome-settings"/>
+	</menu>
+
+	<separator/>
+
+	<!--
+	  Następna linia zawiera autogenerowane mnue na obecnym poziomie.
+	  Jeśli chcesz, możesz zamieścić to we własnym podmenu.
+	-->
+	<include type="system" style="simple" unique="true"/>
+
+	<!--
+	  Odkomentuj poniższą linie (i zakomentuj powyższą) jeśli wolisz raczej
+	  zawrzeć oddzielny plik z menu zamiast używać autogenerowanego menu.
+	-->
+	<!-- <include type="file" src="/etc/xdg/xfce4/desktop/menu2.xml"/> -->
+	
+    <separator/>
+    
+    <app name="Pomoc" cmd="xfhelp4" icon="gnome-help"/>
+    <app name="O Xfce" cmd="xfce4-about" icon="gnome-info"/>
+    <!--
+      Poniższe spowoduje zakończenie xfce4-session, po uprzednim
+      wyświetleniu dialogu "Log out".  Jeśli xfce4-session nie jest
+      uruchomione, spowoduje zakończenie xfdesktop.
+    -->
+    <builtin name="Zakończ" cmd="quit" icon="gnome-logout"/>
+
+</xfdesktop-menu>
