--- ./configure.ac.org	2008-10-28 12:47:03.594776250 +0100
+++ ./configure.ac	2008-10-28 12:56:50.551242090 +0100
@@ -13,11 +13,9 @@ m4_define([exo_minimum_version], [0.3.2]
 m4_define([thunar_minimum_version], [0.8.0])
 m4_define([dbus_minimum_version], [0.34])
 m4_define([intltool_minimum_version], [0.31])
-m4_define([autoconf_minimum_version], [2.50])
 
 dnl init autoconf
 AC_INIT([xfdesktop], [xfdesktop_version], [http://bugzilla.xfce.org/])
-AC_PREREQ([autoconf_minimum_version])
 
 dnl init automake
 XFDESKTOP_VERSION=xfdesktop_version
