diff -urN xfdesktop-4.4.0/menueditor/xfce-menueditor.desktop.in xfdesktop-4.4.0.new/menueditor/xfce-menueditor.desktop.in
--- xfdesktop-4.4.0/menueditor/xfce-menueditor.desktop.in	2007-01-20 22:18:51.000000000 +0100
+++ xfdesktop-4.4.0.new/menueditor/xfce-menueditor.desktop.in	2007-04-03 19:09:06.000000000 +0200
@@ -6,6 +6,6 @@
 Icon=xfce4-menueditor
 Terminal=false
 Type=Application
-Categories=X-XFCE;Application;Settings;DesktopSettings;
+Categories=X-XFCE;Settings;DesktopSettings;
 OnlyShowIn=XFCE;
 _GenericName=Menu Editor
