diff -urN xfdesktop.orig/configure.ac xfdesktop/configure.ac
--- xfdesktop.orig/configure.ac	2004-08-15 01:06:20.000000000 +0200
+++ xfdesktop/configure.ac	2004-08-16 23:03:36.156724472 +0200
@@ -32,8 +32,8 @@
 AC_CHECK_FUNCS(mmap sigaction srandom)
 
 dnl Check for i18n support
-BM_I18N([xfdesktop], [ar az bg ca cs de es es_MX eu fa_IR fi fr he hi hu
-		      it ja pl ko lt ms nl pt_BR pt_PT ru sv ta tr uk vi
+BM_I18N([xfdesktop], [ar az bg ca cs de es es_MX eu fa fi fr he hi hu
+		      it ja pl ko lt ms nl pt_BR pt ru sv ta tr uk vi
 		      zh_CN zh_TW])
 
 dnl Check for X11 installed
