--- xfdesktop-4.1.90/configure.ac.orig	2004-09-26 12:42:32.000000000 +0200
+++ xfdesktop-4.1.90/configure.ac	2004-10-08 07:04:22.862382424 +0200
@@ -33,8 +33,8 @@
 AC_CHECK_FUNCS(mmap sigaction srandom)
 
 dnl Check for i18n support
-BM_I18N([xfdesktop], [ar az be bg ca cs de es es_MX et eu fa_IR fi fr gu he hi
-		      hu it ja pl ko lt ms nl pt_BR pt_PT ru sk sv ta tr uk vi
+BM_I18N([xfdesktop], [ar az be bg ca cs de es es_MX et eu fa fi fr gu he hi
+		      hu it ja pl ko lt ms nl pt_BR pt ru sk sv ta tr uk vi
 		      zh_CN zh_TW])
 
 dnl Check for X11 installed
--- xfdesktop-4.1.90/Makefile.am.orig	2004-10-08 07:16:03.558860368 +0200
+++ xfdesktop-4.1.90/Makefile.am	2004-10-08 07:16:32.664435648 +0200
@@ -39,7 +39,7 @@
 	menu.xml.de 							\
 	menu.xml.es							\
 	menu.xml.eu							\
-	menu.xml.fa_IR 							\
+	menu.xml.fa 							\
 	menu.xml.fr 							\
 	menu.xml.he							\
 	menu.xml.hu 							\
